web-ng/src/app/services/auth/auth.service.ts

```diff
@@ -76,4 +76,24 @@ export class AuthService {
     await this.afAuth.signOut();
     return this.router.navigate(['/']);
   }
+
+  /**
+   * Returns a fresh JSON Web Token (JWT) used to identify the user to a
+   * Firebase service.
+   */
+  private async getIdToken(): Promise<string> {
+    const user = await this.afAuth.currentUser;
+    return await user!.getIdToken(/* forceRefresh */ true);
```

Comment:
Just tested on hosted Firebase and this is returning `error="invalid_token" error_description="The access token could not be verified"`. Trying to debug this but running out of leads. Please hold off on merging for now.

Reply:
Correction: the server is replying with this error in the "Authorization" response header.

Reply:
This also appears relevant:
https://firebase.google.com/docs/auth/admin/manage-cookies#verify_session_cookie_and_check_permissions

Reply:
And here:
https://firebase.google.com/docs/auth/web/auth-state-persistence#modifying_the_auth_state_persistence